package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@radix-ui/react-avatar": "^1.0.4",
+    "@radix-ui/react-dialog": "^1.0.5",
```

Comment:
This is causing the issue. keep both of them as required

Reply:
alright its done @ashutosh-rath02